cilium/CFP-30984-dns-proxy-ha.md

```diff
@@ -94,6 +97,34 @@ Since policy revision numbers are reset when agent restarts, we need to uncondit
 
 ## Impacts / Key Questions
 
+### Getting the DNS Rules
```

Comment:
I think this section is now decently flushed out. I'm thinking we can move it from `Impacts / Key Questions` to the main proposal. We now have an agreement that this is how SDP should discover rules right.